src/Aquifer.API/Endpoints/Help/Endpoint.cs

```diff
@@ -0,0 +1,44 @@
+using Aquifer.Data;
+using Aquifer.Data.Entities;
+using FastEndpoints;
+using Microsoft.EntityFrameworkCore;
+
+namespace Aquifer.API.Endpoints.Help;
+
+public class Endpoint(AquiferDbContext dbContext) : EndpointWithoutRequest<Response>
+{
+    public override void Configure()
+    {
+        Get("/help");
+    }
+
+    public override async Task HandleAsync(CancellationToken ct)
+    {
+        var response = new Response
+        {
+            Releases = await GetRelesesAsync(ct),
+            HowTos = await GetHowTosAsync(ct)
+        };
+        await SendOkAsync(response, ct);
+    }
+
+    private async Task<List<HelpDocumentEntity>> GetRelesesAsync(CancellationToken ct)
+    {
+        return await dbContext
+            .HelpDocuments
+            .Where(x => x.Type == HelpDocumentType.Release)
+            .OrderByDescending(x => x.Updated)
+            .Take(10)
+            .ToListAsync(ct);
+    }
+
+    private async Task<List<HelpDocumentEntity>> GetHowTosAsync(CancellationToken ct)
+    {
+        return await dbContext
+            .HelpDocuments
+            .Where(x => x.Type == HelpDocumentType.HowTo)
+            .OrderByDescending(x => x.Updated)
+            .Take(10)
```

Comment:
Was this in the requirements? For Releases a limit makes sense but wouldn't we want to show all How-Tos?

Reply:
I'll remove the limit on the how top